.github/workflows/smoketest.yaml

```diff
@@ -78,6 +78,13 @@ jobs:
           k0sRealPath="$(realpath "k0s-$K0S_VERSION")"
           echo K0S_UPDATE_FROM_PATH="$k0sRealPath" >>"$GITHUB_ENV"
 
+      - name: "Cache :: GOMODCACHE"
```

Comment:
Since the k0s repo surpasses the default cache limit of 10 GiB almost all of the time, I was thinking about removing a lot of less important caches. In particular, the go modules caching doesn't bring that much of a speedup, does it?

Reply:
I was trying to optimize the tests a bit, but looking at it, it's not that much of an improvement.

I also had a quick look at the bootloose-alpine alpine that's fairly quick too so I guess it probably makes sense to just close this.